trace/context.go

```diff
@@ -48,18 +47,23 @@ func GetIDFromContext(ctx context.Context) ID {
 	return GetTraceFromContext(ctx).ID
 }
 
-//
-// [DEPRECATED]
-//
+// WithTraceAndLabels returns the @parent context with the Trace @trace
+// and the labels @labels
+func WithTraceAndLabels(parent context.Context, trace Trace, labels map[string]string) context.Context {
```

Comment:
Some idea for the future. Create the tye `Labes`, just like the http has the `Values` type: https://golang.org/pkg/net/url/#Values